dnf-behave-tests/features/repoquery/globs.feature

```diff
@@ -0,0 +1,178 @@
+Feature: Glob tests for expanding all the various glob patterns.
+
+Background:
+ Given I use the repository "repoquery-globs"
+
+
+# <name> globs
+Scenario: repoquery '*'
+ When I execute dnf with args "repoquery '*'"
+ Then the exit code is 0
+  And stdout is
+      """
+      desktop-utils-1:1.0-1.src
+      desktop-utils-1:1.0-1.x86_64
+      desktop-utils-1:1.23.9-1.src
+      desktop-utils-1:1.23.9-1.x86_64
+      htop-1:1.0-1.src
+      htop-1:1.0-1.x86_64
+      top-1:1.0-1.src
+      top-1:1.0-1.x86_64
+      toped-1:1.0-1.src
+      toped-1:1.0-1.x86_64
+      topgit-1:1.0-1.src
+      topgit-1:1.0-1.x86_64
+      topgit-1:1.17.6-1.src
+      topgit-1:1.17.6-1.x86_64
+      toppler-1:1.0-1.src
+      toppler-1:1.0-1.x86_64
+      """
+
+Scenario: repoquery top*
+ When I execute dnf with args "repoquery top*"
+ Then the exit code is 0
+  And stdout is
+      """
+      top-1:1.0-1.src
+      top-1:1.0-1.x86_64
+      toped-1:1.0-1.src
+      toped-1:1.0-1.x86_64
+      topgit-1:1.0-1.src
+      topgit-1:1.0-1.x86_64
+      topgit-1:1.17.6-1.src
+      topgit-1:1.17.6-1.x86_64
+      toppler-1:1.0-1.src
+      toppler-1:1.0-1.x86_64
+      """
+
+Scenario: repoquery top?d
+ When I execute dnf with args "repoquery top?d"
+ Then the exit code is 0
+  And stdout is
+      """
+      toped-1:1.0-1.src
+      toped-1:1.0-1.x86_64
+      """
+
+Scenario: repoquery top?{d,it}
+ When I execute dnf with args "repoquery top?{{d,it}}"
```

Comment:
Why there are two {{?

Reply:
It's because `format()` is called on the argument in the steps. The way to escape `{}` for `format()` is to double them.